<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>270</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>270</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>480</width>
    <height>270</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>22</y>
     <width>400</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLineEdit" name="lineEdit_Username">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>220</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>             请输入账号</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Password">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>220</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>             请输入密码</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Password_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>130</y>
      <width>220</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>16</number>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>             再输入密码</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Register">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>180</y>
      <width>220</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>注册</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>210</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>管理员</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
